sqlbrite-lint/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'kotlin'
+
+dependencies {
+  compileOnly rootProject.ext.kotlinStdLib
+  compileOnly rootProject.ext.lintApi
+  compileOnly rootProject.ext.lintChecks
+
+  testCompile rootProject.ext.junit
+  testCompile rootProject.ext.lint
+  testCompile rootProject.ext.lintTests
```

Comment:
`testCompile` => `testImplementation` now that this project is using Android Gradle Plugin 3.0.